### ISSUE TITLE
Consistent formatting for chemical naming

### ISSUE THREAD

**Issue by jump2conclusionsmatt** [USEPA/Federal-LCA-Commons-Elementary-Flow-List#53]
I'll admit I'm pretty ignorant here. Should there be a consistent naming convention for things like Benzo(b)fluoranthene and Benzo[j]fluoranthene? Should they both use parentheses or both use brackets?

**Comment by ashleyedelen** [comment]
So, the inconsistency in the naming is due to the inconsistencies that exist within the EPA databases (SRS and Chemistry Dashboard) that we use to define our chemicals and groups. When we first started building the list we noticed that these sources are not consistent in the formating. We do hope to go back and do some curration at some point and then reach out to the EPA databases to discuss their inconsistencies, but at this time we have not gone through this process. Thanks for the reminder, this is an issue we hope to resolve as we further currate our list.
